### PR TITLE
fix invisible line between labels

### PR DESCRIPTION
**Pull request by larseirikhansen**

### CODE REVIEW

**Review by KenAJoh — APPROVED** [review]

File: @navikt/core/css/stepper.css [hunk 1]
```diff
@@ -95,5 +95,5 @@
   border-left: 2px solid var(--navds-semantic-color-border-muted);
   border-radius: var(--navds-border-radius-medium);
   pointer-events: none;
-  z-index: -1;
+  z-index: 1;
```
Comment (andnorda) [inline @navikt/core/css/stepper.css:98]: Funker dette uten `z-index: 1;`?
```suggestion
```

Comment (KenAJoh) [reply @navikt/core/css/stepper.css:98]: Uten z-index: 1
Screenshot 2022-04-05 at 10 32 12

Med z-index: 1
Screenshot 2022-04-05 at 10 32 18

Comment (andnorda) [reply @navikt/core/css/stepper.css:98]: Ah, skjønner!

**Review by andnorda — APPROVED** [review]
LGTM

**Pull request merged**